dev-tools/packaging/templates/docker/Dockerfile.tmpl

```
@@ -8,7 +8,15 @@ FROM {{ .buildFrom }} AS home
 
 COPY beat {{ $beatHome }}
 
-RUN mkdir {{ $beatHome }}/data {{ $beatHome }}/logs && \
+{{- if (and (eq .BeatName "heartbeat") (not (contains .from "ubi-minimal"))) }}
+RUN mkdir -p {{ $beatHome }}/.node \
+    {{ $beatHome }}/.npm \
+    {{ $beatHome }}/.cache \
+    {{ $beatHome }}/.config \
+    {{ $beatHome }}/suites && \
```

There is an error here, that is making builds to fail also in master, could you please take a look?
```suggestion
    {{ $beatHome }}/suites
```

---

I have opened a PR to detect these issues that includes this fix: https://github.com/elastic/beats/pull/23401